### DISCUSSION TITLE
Disabling replicas completely for a longhorn volume

### DISCUSSION THREAD

**Discussion by krishnakc1** [longhorn/longhorn#7414] [Q&A]
Is it possible to completely disable replicas for a particular longhorn volume(s)? I checked the settings on longhorn UI and the default replicas should be between 1 and 20. I set the replicas at storage class level as well to 0 but it still does not take effect. Is there some other place where this could be set? I want to disable volume replication because I have a non prod server and the volumes can be as big as 500G for which I have no use of even if I lose that data. Thank you.

**Comment by mantissahz** [comment]
If you set the replica number 1, this replica is only one place to store the data.
That would be like that if you set replica number 2, that is one place stores the data and another one is a replica of the volume.

**Reply by krishnakc1** [reply]
Thanks for the response. However, I do not even want to maintain replica of a volume. Is it possible to do this via longhorn?

**Reply by christiaan-bronkhorst** [reply]
I have the same question, is this even possible?
